tests/specification/deepCopy.js

```diff
@@ -1,9 +1,9 @@
+/* eslint-disable comma-dangle */
 import test from "ava";
 import { deepCopy, deepCopyAdded } from "../../deep.js";
 import { primitives } from "../helper.js";
 
-
-const runBaselineDeepCopyTests = (deepCopyImplementation) => {
+const runBaselineDeepCopyTests = deepCopyImplementation => {
```

Comment:
2 blank lines after exports/imports

Reply:
Fixed that in recent commit

Reply:
Did you git push ?

Reply:
Yeah I did.

Reply:
Last commit I see is from 9 hours ago, before I made the comment
![image](https://user-images.githubusercontent.com/5721194/73864380-d1e8f200-4841-11ea-9ea5-205e1959af26.png)

Reply:
Wow. I'll look into it. Sorry for the mix up